src/utils/editor/expression.js

```diff
@@ -64,6 +64,11 @@ export function previewExpression({
   }
 
   if (variables.has(tokenText)) {
+    let variableKey = variables.get(tokenText);
+    if (!get(variableKey, "contents.value.type") == "undefined") {
+      return null;
+    }
+
```

Comment:
the next step is to look at the expression and see if its value is undefined too :)

don't worry, we'll be cleaning this code up soon :P